### PR TITLE
O3-1393 Creating modal to change queue entry status : added edit button

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
-Added edit button on queue table to launch changing status dialog

## Screenshots

https://user-images.githubusercontent.com/19533785/179192704-bfaa93b6-9a7b-4216-9377-367019d0d85d.mp4

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by CynthiaKamau** [comment]
Kindly review : @donaldkibet @makombe

File: packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx [hunk 1]
```diff
@@ -114,6 +115,37 @@ function ActionsMenu({ patientUuid }: { patientUuid: string }) {
...
+      queueEntryUuid,
+      queueUuid,
+    });
+  }, [patientUuid, queueEntryUuid, queueUuid]);
+
+  return (
+    <Button
+      kind="ghost"
+      onClick={launchEditPriorityModal}
+      iconDescription={t('editIdentifierTooltip', 'Edit')}
```
Comment (makombe) [inline packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx:141]:
```suggestion
      iconDescription={t('editQueueEntryStatusTooltip', 'Edit')}
```

File: packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx [hunk 2]
```diff
@@ -114,6 +115,37 @@ function ActionsMenu({ patientUuid }: { patientUuid: string }) {
...
+  patientUuid,
+  queueUuid,
+  queueEntryUuid,
+}: {
+  patientUuid: string;
+  queueEntryUuid: string;
+  queueUuid: string;
+}) {
+  const { t } = useTranslation();
+  const launchEditPriorityModal = useCallback(() => {
```
Comment (makombe) [inline packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx:128]: I think you can change the naming of the modal here to reflect what we are editing, is it a priority modal we are launching or status modal or is it for both priority and status?

**Review by makombe — COMMENTED** [review]
@CynthiaKamau Thanks for the work

File: packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx (hunk 2 above)
Comment (CynthiaKamau) [reply packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx:128]: Done

**Review by makombe — APPROVED** [review]

**Comment by gracepotma** [comment]
CC @slubwama

**Pull request merged**